### PR TITLE
RPI: add handling for InverterDevInform_All message and others

### PR DESCRIPTION
**Pull request by chehrlic**
... so we can retrieve the firmware version on startup. Other stuff may be added later.
Also handle answer for RealTimeRunData_Reality  similar to RealTimeRunData_Debug

### CODE REVIEW

**Comment by stefan123t** [comment]
@chehrlic since you are currently the sole maintainer of the tools/rpi tree I am simply merging your changes. I do not know if you want a peer review you might ask @Sprinterfreak as he spent quite some time on the RPi code initially.
Anyway your contributions are very welcome !

File: tools/rpi/hoymiles/__init__.py [hunk 1]
```diff
@@ -482,21 +482,20 @@ def compose_esb_packet(packet, mtu=17, **params):
...
     """
-    if not timestamp:
-        timestamp = int(time.time())
+    timestamp = int(time.time())
 
-    payload = b'\x0b\x00'
+    payload = struct.pack('>B', cmdId) + b'\x00'
     payload = payload + struct.pack('>L', timestamp)  # big-endian: msb at low address
-    payload = payload + b'\x00\x00\x00\x05\x00\x00\x00\x00'
+    payload = payload + b'\x00\x00\x00\x00\x00\x00\x00\x00'
```
Comment (stefan123t) [inline tools/rpi/hoymiles/__init__.py:498]: Here should the AlarmMessageId counter be inserted. This lets the Inverter know up till which AlarmId we have seen the log.

**Review by stefan123t — APPROVED** [review]
See inline

File: tools/rpi/hoymiles/__init__.py (hunk 1 above)
Comment (chehrlic) [reply tools/rpi/hoymiles/__init__.py:498]: I'm currently working on this one - I think tomorrow I can add the code to request the alarm request code. Needs some testing but the sun is already down for today.

**Pull request merged**